src/LtiRegistration.php

```diff
@@ -61,7 +61,7 @@ public function getKeySetUrl()
         return $this->keySetUrl;
     }
 
-    public function setKeySetUrl($keySetUrl)
+    public function setKeySetUrl($keySetUrl): self
```

Comment:
The same is true for the remaining `set` functions - should all of the parameters be of type `string`?